GVFS/GVFS.Upgrader/UpgradeOptions.cs

```diff
@@ -0,0 +1,22 @@
+using CommandLine;
+
+namespace GVFS.Upgrader
+{
+    [Verb("UpgradeOrchestrator", HelpText = "Upgrade VFS for Git.")]

```

Comment:
This `Verb` annotation seems out of place.

Reply:
This usage is different than how we generally structure this in our product, but it does match the intended usage:

https://github.com/commandlineparser/commandline#for-verbs

I think we could ask whether we need this to be a verb, as this is being used to process command line arguments for a program that doesn't actually have verbs. I will note this as a potential cleanup.